src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/IViewBufferScope.cs

```diff
@@ -12,9 +12,9 @@ public interface IViewBufferScope
     {
         /// <summary>
         /// Gets a <see cref="T:ViewBufferValue[]"/>.
-        /// </summary>
+        /// <param name="size">The minimum size of the returned segment.</param>
```

Comment:
- Restore `</summary>`.
- Why does the `size` description say "returned" for a "Get" method?
- Should this parameter be named `pageSize`?